Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml

```diff
@@ -26,6 +26,7 @@
     sprintModifier: 0.80
   - type: ToggleableClothing
     clothingPrototype: ClothingHeadHelmetHardsuitBasic
+    slot: head
```

Comment:
if you add that to parent you don't need to use it again in prototypes that uses that as parent